package.json

```diff
@@ -75,18 +75,20 @@
     "ngx-device-detector": "^1.3.0",
     "ngx-md": "~6.0.0",
     "ngx-pagination": "^3.0.3",
+    "node-sass": "^4.9.3",
```

Comment:
This looks like a development dependency, I don't think it should have been added here?